### PR TITLE
Show deltas for FTEs & total positions in central program pages

### PR DESCRIPTION
**Pull request by emilybram**
Example:

image

Also sidenote, Prettier reformatted a bunch of stuff I didn't touch, I can highlight the code that actually did change in this PR.

### CODE REVIEW

File: src/components/central-program.js [hunk 1]
```diff
@@ -108,6 +108,10 @@ export const query = graphql`
         description
         abbreviation
       }
+      change_from_previous_year {
```
Comment (emilybram) [inline src/components/central-program.js:111]: This is the part I changed in this file

File: src/components/central-program/program-data-overview-table.js [hunk 2]
```diff
@@ -125,30 +141,57 @@ const BudgetOverview = ({data}) => {
...
+      align: "right",
+      headerFormatter: (column, colIndex, components) => {
+        return (
+          <div className="text-right value">{data.eoy_total_positions}</div>
+        )
+      },
+    },
   ]
 
+  const rows = []
```
Comment (emilybram) [inline src/components/central-program/program-data-overview-table.js:165]: This is the part I changed in this file

File: src/components/table-utilities.js [hunk 3]
```diff
@@ -1,44 +1,70 @@
...
+  }
   return 0
 }
 
-
-
 // Convert null to zero & Round without trailing zeroes
 export const formatFTE = fte => (!fte ? 0 : +fte.toFixed(2))
+
+// Construct a prefix to add to the start of a number that is meant to represent a delta.
```
Comment (emilybram) [inline src/components/table-utilities.js:63]: This is the part I changed in this file

File: src/components/central-program/program-data-overview-table.js [hunk 4]
```diff
@@ -125,30 +141,57 @@ const BudgetOverview = ({data}) => {
...
+
+  if (data.change_from_previous_year) {
+    const total_positions_delta =
+      data.change_from_previous_year.eoy_total_positions
+
+    rows.push({
+      description: "Change in staff from previous year",
```
Comment (jbaldo) [inline src/components/central-program/program-data-overview-table.js:172]:
```suggestion
      description: "Change in total positions from previous year",
```

**Review by jbaldo — COMMENTED** [review]
nice! I like your suggestion of putting Total positions on its own line.

Realizing that we should put in this same footnote about staff data but it come come later if it's a pain. https://github.com/openoakland/openousd-site/blob/master/src/components/central-programs-table.js#L291

**Comment by emilybram** [comment]
@jbaldo I've put staff in a separate row (screenshot updated in PR) but I'm having trouble putting the material design arrows into the table -- since the `rows` field passed in to `ToolkitProvider` (a bootstrap component) have a `value` property that's supposed to be a string. Since the arrows are separate, non-text components that would need to be rendered, I'm not sure how to make it work with the `ToolkitProvider` component. Do you have any thoughts here?

**Pull request merged**